src/widgets/immunizations/immunization-mapper.ts

```diff
@@ -0,0 +1,111 @@
+import { get, groupBy, map, orderBy } from "lodash-es";
+import dayjs from "dayjs";
+
+const mapToImmunizationDoses = immunizationResource => {
```

Comment:
Can we type this? What is `immunizationResource`?

Reply:
It is the FHIR immunization resource. I should be able to resue the one which is sent to server.